src/SDKs/EventHub/Management.EventHub/Microsoft.Azure.Management.EventHub.csproj

```diff
@@ -18,6 +18,10 @@
 		</PackageReleaseNotes>
 		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
 	</PropertyGroup>
+  <PropertyGroup>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
@v-Ajnava all comments apply to this PR as well that I have provided in #3948 
for both the projects

Reply:
Resolved

Reply:
I still see it? line 22